lib/i18n/english/tags/blog-post.js

```diff
@@ -0,0 +1,36 @@
+export default {
+      "noPostText":"This post is no longer available!",
+      "editLinkText":"Edit",
+      "publishLinkText" : "Publish",
+      "unpublishLinkText" : "Unpublish",
+      "deleteLinkText":"Delete",
+      "saveLinkText" : "Save",
+      "unknownUserText":"Unknown User",
+      "replyText":"reply",
+      "repliesText":"replies",
+      "replyComposerText":"Write a Reply",
+      "replyInputPlaceholder":"Write your reply here...",
+      "replyButtonText" : "Send Reply",
+      "clearButtonText" : "Clear",
+      "loginButtonText" : "Login to write a reply",
+      "publishProgressText" : "Publishing...",
+      "publishErrorText" : "Couldn't be published!",
+      "unpublishProgressText" : "unPublishing...",
```

Comment:
"unpublishProgressText" : "unPublishing..."

This is wrong. Why are you using camelCase for the text that will be read by end user. "unPublishing..." Should be "Unpublishing..."